src/test/java/by/exadel/internship/formTest/FormTest.java

```diff
@@ -60,15 +62,18 @@ public void givenFormWithFile_checkTestData()
         assertEquals(formFullDTO.getSkype(), "string");
     }
 
-    @Test
-    public void checkListSize() throws Exception {
```

Comment:
What's wrong with test?

Reply:
We changed endpoint(appended parametr internshipId) It will be redone soon